src/localization/fi.json

```diff
@@ -221,7 +222,12 @@
         "formVersion": "Versio",
         "formStyling": "Lomakkeen ulkoasu",
         "backgroundImage": "Taustakuva",
-        "backgroundColor": "Taustaväri"
+        "backgroundColor": "Taustaväri",
```

Comment:
These two strings could probably be removed since they're not used anymore (lines 224-225)

Reply:
Done